test/verify/check-machines-virsh

```diff
@@ -17,26 +17,26 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
+import subprocess
+import unittest
+
 import machineslib
 import parent
 from testlib import *
 
 
-@skipImage("LibvirtDBus is the only available provider",
-           "fedora-30", "fedora-i386", "fedora-testing", "rhel-8-0", "rhel-8-0-distropkg", "rhel-8-1")
 class TestMachinesVirsh(machineslib.TestMachines):
-
-    def removeLibvirtDBus(self):
-        m = self.machine
-        if ("debian" in self.machine.image) or ("ubuntu" in self.machine.image):
-            m.execute("dpkg --remove libvirt-dbus || true")
-        else:
-            m.execute("rpm -e --nodeps libvirt-dbus || true")
+    def skipIfLibvirtDbusAvailable(self):
+        try:
+            self.machine.execute("busctl introspect org.libvirt /")
+            raise unittest.SkipTest("libvirt-dbus is available; not testing virsh backend")
+        except subprocess.CalledProcessError:
+            pass
```

Comment:
As discussed on IRC -- let's please not make this dynamic, it will hide errors if e. g. libvirt-dbus is broken or it accidentally falls off an image. Please make it a static list of OSes, i. e. test virsh on ubuntu-1804, and libvirt-dbus everywhere else.

Reply:
Done